src/auth/AxiosJwtAuthService.js

```diff
@@ -0,0 +1,303 @@
+import axios from 'axios';
+import PropTypes from 'prop-types';
+import { logFrontendAuthError } from './utils';
+import { camelCaseObject, ensureDefinedConfig } from '../utils';
+import createJwtTokenProviderInterceptor from './interceptors/createJwtTokenProviderInterceptor';
+import createCsrfTokenProviderInterceptor from './interceptors/createCsrfTokenProviderInterceptor';
+import createProcessAxiosRequestErrorInterceptor from './interceptors/createProcessAxiosRequestErrorInterceptor';
+import AxiosJwtTokenService from './AxiosJwtTokenService';
+import AxiosCsrfTokenService from './AxiosCsrfTokenService';
+
+const optionsPropTypes = {
+  config: PropTypes.shape({
```

Comment:
Note, we've changed the options signature here.  This is _technically_ a breaking change, but I'm consciously not marking it as such because no dependents of this project rely on `configure` directly - they all have it done for them via `initialize`.